apps/voting/test/voting.js

```diff
@@ -318,7 +418,7 @@ contract('Voting App', accounts => {
         })
 
         it('creating vote as holder executes vote', async () => {
```

Comment:
I would change this description, as it's no longer the case when creating a "regular" vote.